osu.Game/Overlays/Rankings/Tables/UserBasedTable.cs

```diff
@@ -78,7 +79,7 @@ public GradeHeaderText(LocalisableString text, bool isHighlighted)
                 {
                     // Grade columns have extra horizontal padding for readibility
                     Horizontal = 20,
-                    Vertical = 5
+                    Vertical = 150
```

Comment:
What's the reason for increasing this?

Reply:
There is none. This is just plain wrong.

Reply:
oops that is unintentional.